core/src/sigverify.rs

```diff
@@ -88,49 +47,24 @@ impl SigVerifier for TransactionSigVerifier {
     #[inline(always)]
```

Comment:
can we just take the default trait implementation for these (besides send of course)?

Reply:
Even more dead lines!

Reply:
Used the default implementations: cdaa1b02320be1497fc6045bd6c672f7e9a10797

Then realized, now that THIS isn't implementing those functions...nothing is.
fc49d453b17dd52270748898bef416add54f7a30, fb0ab3242c0491f1e8e64528f8f8281e5d474dca, 9a23f07ebd7ce2f5cf81aafd190816cfcc382a37
Let us remove an additional ~100 lines of unused code.